.github/workflows/awie.yml

```diff
@@ -65,6 +65,8 @@ jobs:
     needs: [get-version, package]
     if: ${{ contains(fromJson('["stable", "testing", "unstable"]'), needs.get-version.outputs.stability) }}
     runs-on: [self-hosted, common]
+    environment: ${{ needs.get-version.outputs.environment }}
```

Comment:
I'm wondering if it'd make sense to add the version in the environment? because it might be useful for testers to know to what repo as well as the corresponding version the ticket has been deployed

Reply:
Env name looks like `XX.YY-stability` as asked initially.
The output comes from the `get-version` action. It's built from the Major version and Stability values.
I just checked it again, the minor version appears in the env name. I'll fix that in the next commit.

Reply:
well if it's the case, there is nothing to do :)